frontend/test/metabase/scenarios/binning/correctness/time-series.cy.spec.js

```diff
@@ -27,7 +27,10 @@ describe("scenarios > binning > correctness > time series", () => {
 
     cy.createQuestion(questionDetails, { visitQuestion: true });
 
-    cy.findByText("Summarize").click();
+    cy.findByTestId("qb-header-action-panel")
+      .contains("Summarize")
+      .click();
```

Comment:
Wouldn't this be the same, just more direct?
```suggestion
    cy.findByTestId("toggle-summarize-sidebar-button").click();
```

Reply:
There is some race condition and I can't extract the exact moment when it occurs. Some other tests are affected as well, but it really depends on the speed of the runner. Newer helper will take care of that.

`cy.findByTestId("toggle-summarize-sidebar-button").click();` should work in theory, but if you query the document for that testid, you'll actually see two elements. The other one is the button that we trigger for the mobile view. It's rendered in the DOM, only hidden.
That's why your suggested approach fails and I had to use the workaround.